### PR TITLE
Fix part of #2834: Lambda function descriptive naming in PromotedStoryViewModelTest

### PR DESCRIPTION
**Pull request by jonathanalvares9009**
## Explanation
* Fixes part of #2834: Lambda function descriptive naming
* This makes it more readable to future developers working on the project
 
## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/sharedTest/java/org/oppia/android/app/home/promotedlist/PromotedStoryViewModelTest.kt
Comment (rt4914) [inline app/src/sharedTest/java/org/oppia/android/app/home/promotedlist/PromotedStoryViewModelTest.kt]: You should not name both the lambda functions same.

For above use `homeFragmentTestActivityScenario` and for internal use `homeFragmentTestActivity`

Comment (rt4914) [reply app/src/sharedTest/java/org/oppia/android/app/home/promotedlist/PromotedStoryViewModelTest.kt]: Ditto else where.

Something like this:
```
    ).use { homeFragmentTestActivityScenario ->
      homeFragmentTestActivityScenario.onActivity { homeFragmentTestActivity ->
        val promotedStoryViewModel = createBasicPromotedStoryViewModel(homeFragmentTestActivity)
```

**Review by rt4914 — CHANGES_REQUESTED** [review]
Thanks for the PR.
Make sure once you do the changes you reply to the comments.

Also, update your title to mention the issue number like `Fix #XXXX: Lambda function descriptive naming`

**Comment by jonathanalvares9009** [comment]
@rt4914 made the changes. N please excuse me if I made any errors I'm new to open source trying to acquaint myself with it.

File: app/src/sharedTest/java/org/oppia/android/app/home/promotedlist/PromotedStoryViewModelTest.kt
Comment (rt4914) [reply app/src/sharedTest/java/org/oppia/android/app/home/promotedlist/PromotedStoryViewModelTest.kt]: @jonathanalvares9009 can you please reply to this.

**Comment by rt4914** [comment]
> @rt4914 made the changes. N please excuse me if I made any errors I'm new to open source trying to acquaint myself with it.

No issues. Errors/mistakes are part of learning. Can you please update PR title as per this comment: https://github.com/oppia/oppia-android/pull/2873#pullrequestreview-608797661 @JonathanAlvares9009

**Comment by jonathanalvares9009** [comment]
Changed the pr title as told @rt4914

**Comment by MaskedCarrot** [comment]
@jonathanalvares9009 The pr title had to be updated as per [this comment](https://github.com/oppia/oppia-android/pull/2873#pullrequestreview-608797661) so please update the PR title to `Fix part of #2834: Lambda function descriptive naming in PromotedStoryViewModelTest`

**Comment by anandwana001** [comment]
@jonathanalvares9009 Please follow the checklist and mark it only if it completed.

```
[ ] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
[ ] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
```

File: app/src/sharedTest/java/org/oppia/android/app/home/promotedlist/PromotedStoryViewModelTest.kt
Comment (jonathanalvares9009) [reply app/src/sharedTest/java/org/oppia/android/app/home/promotedlist/PromotedStoryViewModelTest.kt]: @rt4914 done that and sent a pull request

**Comment by jonathanalvares9009** [comment]
Sorry @anandwana001 will follow it next time.

**Comment by anandwana001** [comment]
@jonathanalvares9009 I had re-run the checks, let's see if it passes. Please fix the second point of the checklist. 
Also, the PR doesn't merge automatically, once all the check passes, you can mention me here to review this PR again.

**Comment by jonathanalvares9009** [comment]
@anandwana001 I have made the changes can you review it?

**Comment by anandwana001** [comment]
@jonathanalvares9009 Please follow 
```
The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
```

**Review by anandwana001 — APPROVED** [review]
LGTM, Please correct the second point from the checklist.

**Comment by jonathanalvares9009** [comment]
@anandwana001 I have made the changes please review them. Thank you for having patience with me.

**Comment by jonathanalvares9009** [comment]
@anandwana001 is there still an issue with the PR.

**Comment by anandwana001** [comment]
No, it's good to go now, need approval from Rajat as he is the code owner.

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**
